### PR TITLE
CVE-2022-21724: hmc-cft-hearing-service fix

### PR DESCRIPTION
**Pull request by duranmf**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-3113

### Change description ###

pgjdbc is the offical PostgreSQL JDBC Driver. A security hole was found in the jdbc driver for postgresql database while doing security research. The system using the postgresql library will be attacked when attacker control the jdbc url or properties. pgjdbc instantiates plugin instances based on class names provided via `authenticationPluginClassName`, `sslhostnameverifier`, `socketFactory`, `sslfactory`, `sslpasswordcallback` connection properties. However, the driver did not verify if the class implements the expected interface before instantiating the class. This can lead to code execution loaded via arbitrary classes. Users using plugins are advised to upgrade. There are no known workarounds for this issue.

upgraded to --> group: 'org.postgresql', name: 'postgresql', version: '42.3.2'

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/hmc/utils/TestingUtil.java [hunk 1]
```diff
@@ -72,6 +72,7 @@
 import java.time.LocalTime;
 import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
+import java.util.Arrays;
```
Comment (MSancaktutar) [inline src/test/java/uk/gov/hmcts/reform/hmc/utils/TestingUtil.java:75]: why additional import?

File: src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingActualsResponseMapper.java [hunk 2]
```diff
@@ -43,33 +43,6 @@ public HearingActualResponse toHearingActualResponse(HearingEntity hearingEntity
         return response;
     }
 
-    private String getHearingStatus(HearingEntity hearingEntity) {
```
Comment (MSancaktutar) [inline src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingActualsResponseMapper.java:46]: As this app is still work in progress, better remove `private` then the method itself, if the method is unused.

File: src/test/java/uk/gov/hmcts/reform/hmc/utils/TestingUtil.java (hunk 1 above)
Comment (duranmf) [reply src/test/java/uk/gov/hmcts/reform/hmc/utils/TestingUtil.java:75]: it is not an additional import, it was missing, i have no idea how it was being compiled.

**Review by MSancaktutar — APPROVED** [review]
lgtm

File: config/owasp/suppressions.xml [hunk 3]
```diff
@@ -26,19 +26,9 @@
     </notes>
```
Comment (rajaram-nagarajan) [inline config/owasp/suppressions.xml:26]: Can you correct the suppress date for CVE-2018-1258?

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**